src/engine/property_descriptor.h

```diff
@@ -538,16 +539,21 @@ class ResourcePropertyDescriptor : public FilePropertyDescriptor<T>,
 								   public ResourcePropertyDescriptorBase
 {
 public:
+	using Getter = typename FilePropertyDescriptor<T>::Getter;
```

Comment:
GCC do not lookup into template base classes for name resolution (http://stackoverflow.com/questions/4010281/accessing-protected-members-of-superclass-in-c-with-templates)